ietf-te-types.yang

```diff
@@ -1429,13 +1428,13 @@ module ietf-te-types {
       "Base identity for TE tunnel administrative states.";
   }
 
-  identity tunnel-admin-state-up {
+  identity up {
```

Comment:
This would be an NBC change so prefer not to implement it. How to differentiate an admin-up from an admin-down?

Reply:
identities are always prefixed; so there is no confusion.

Reply:
OK with NBC comment, though. reverted back.